src/SFA.DAS.EmployerIncentives.Api/Controllers/CollectionCalendarCommandController.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using SFA.DAS.EmployerIncentives.Abstractions.Commands;
+using SFA.DAS.EmployerIncentives.Api.Types;
+using SFA.DAS.EmployerIncentives.Commands.CollectionPeriod;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.EmployerIncentives.Api.Controllers
+{
+    [Route("collectionCalendar")]
+    [ApiController]
+    public class CollectionCalendarCommandController : ApiCommandControllerBase
+    {
+        public CollectionCalendarCommandController(ICommandDispatcher commandDispatcher) : base(commandDispatcher)
+        {
+        }
+
+        [HttpPatch]
+        [Route("period/active")]
```

Comment:
This is just "collectionPeriods"  the operation is a patch and it is an instance that is being updated.  The data contains the change not the endpoint.